### DISCUSSION TITLE
Ability to set brightness shortcut to the Fn brightness key.

### DISCUSSION THREAD

**Discussion by LupusHacker** [xanderfrangos/twinkle-tray#288] [Ideas]
I've currently set the shortcut to ctrl+up arrow/down arrow, but would rather have it set to one of the FN keys (specifically the shortcut key that controls windows display brightness). Considering it doesn't work on my external display, it only makes sense to "recycle" that shortcut, and make it usable by Twinkle Tray. ^^

**Comment by CadMan77** [comment]
+1 (as a man keen on Lenovo SK-8855 keybords).

**Comment by Tau512** [comment]
this should be optional for laptop users.
obviously brightness will work on the internal laptop screen. connect an external screen as it wont control the brightness.
what i do on my MX Keys keyboard is Fn+f1&2 is for laptop screen. fn+ctrl+f1&f2 for external. Ctrl because it's ajacent to Fn so i can press both keys with 1 finger so it acts like a 2 key press rather than 3.

**Comment by himenohogosha** [comment]
Hello. I just installed this app today and while it works great, I agree with others that it would be nice to support Fn key in the hotkeys. It isn't working as far as I can tell. Thanks.
